.github/workflows/ci.yml

```diff
@@ -14,8 +15,6 @@ jobs:
         run: cargo fmt --all -- --check
       - name: Cargo clippy
         run: cargo clippy --workspace --all-targets -- -D warnings
-      - name: Cargo test
-        run: cargo test --workspace
```

Comment:
Removed this test here as it is already covered by another build step for all targets.